fetch-json/index.html

```diff
@@ -1,50 +1,49 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width" />
 
     <title>Fetch json example</title>
 
-    <link rel="stylesheet" href="style.css">
-    <!--[if lt IE 9]>
-      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
     <h1>Fetch json example</h1>
-    <ul>
-    </ul>
-
+    <ul></ul>
   </body>
   <script>
-    var myList = document.querySelector('ul');
+    const myList = document.querySelector("ul");
+
+    fetch("products.json")
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`HTTP error, status = ${response.status}`);
+        }
+        return response.json();
+      })
+      .then(function (data) {
+        for (const product of data.products) {
+          const listItem = document.createElement("li");
+          
+          const nameElement = document.createElement("strong");
+          nameElement.textContent = product.Name;
 
-    fetch('products.json')
-    .then(function(response) {
-      if (!response.ok) {
-        throw new Error("HTTP error, status = " + response.status);
-      }
-      return response.json();
-    })
-    .then(function(data) {
-      for(var i = 0; i < data.products.length; i++) {
-        var listItem = document.createElement('li');
-        listItem.innerHTML = '<strong>' + data.products[i].Name + '</strong>';
-        listItem.innerHTML +=' can be found in ' + data.products[i].Location + '.';
-        listItem.innerHTML +=' Cost: <strong>£' + data.products[i].Price + '</strong>';
-        myList.appendChild(listItem);
-      }
-    })
-    .catch(function(error) {
-      var p = document.createElement('p');
-      p.appendChild(
-        document.createTextNode('Error: ' + error.message)
-      );
-      document.body.insertBefore(p, myList);
-    });
+          const textNode = document.createTextNode(` can be found in ${product.Location}. Cost: `);
+          
+          const priceElement = document.createElement("strong");
+          priceElement.textContent = `£${product.Price}`;
 
+          listItem.append(nameElement, textNode, priceElement);
```

Comment:
Nit: since we are using `Element.append`—we should be able to avoid the `document.createTextNode` call, right?